app/assets/stylesheets/components/_share-buttons.scss

```diff
@@ -0,0 +1,51 @@
+.share-buttons {
+  text-align: center;
+  padding-top: $base-spacing * 2;
+
+  h4 {
+    font-weight: $font-weight-normal;
+    color: $medium-gray-color;
+  }
+
+  a {
+    display: inline-block;
+    color: $white-color;
+    border: none;
+    padding: 0;
+    width: 3em;
+    height: 3em;
+    line-height: 3em;
+    opacity: .8;
+    margin: 0 ($small-spacing/2);
```

Comment:
`$small-spacing/2` should be written with a single space on each side of the operator: `$small-spacing / 2`